deployment/centos/bootstrap.sh

```diff
@@ -9,6 +9,16 @@ git fetch --tags --all
 git checkout -b v0.2.1 tags/v0.2.1
 echo 'Repo is cloned and ready'
 
+echo 'Create postgresql database and user'
+POSTGRES_USER=${1:-postgres}
+
+psql $POSTGRES_USER -v ON_ERROR_STOP=1 <<-EOSQL
```

Comment:
Postgresql db creation should probably be handled differently than this. Not sure what's best, though.